core/templates/dev/head/pages/exploration_editor/simple_editor_tab/simple_editor_sidebar_directive.html

```diff
@@ -23,9 +23,7 @@
       </li>
     </ul>
 
-    <button type="button" class="btn btn-default full-editor-btn" ng-click="setEditorModeToFull()">
-      Use Advanced Editor
-    </button>
+    <span class="full-editor-switch" ng-click="setEditorModeToFull()">Use Advanced Editor</span>
```

Comment:
Okay, now the button looks like it used to, with 5px up from the bottom of the sidebar.